eva-pipeline/src/main/java/embl/ebi/variation/eva/pipeline/jobs/VariantAnnotConfiguration.java

```diff
@@ -31,10 +31,12 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
 import org.springframework.core.env.Environment;
 
 @Configuration
 @EnableBatchProcessing
+@Import(VariantJobArgsConfig.class)
 public class VariantAnnotConfiguration {
```

Comment:
Oh, I thought the new annotation job was already integrated into the VariantConfiguration class but now I see it's not... Ideally we should test it with a real file and have it merged by the time this ticket is closed. Would you see any reason to postpone it?